custom/conf/app.example.ini

```diff
@@ -1485,39 +1485,36 @@ PATH =
 ;; Prefix displayed before subject in mail
 ;SUBJECT_PREFIX =
 ;;
-;; Mail server
-;; Gmail: smtp.gmail.com:587
-;; QQ: smtp.qq.com:465
-;; As per RFC 8314 using Implicit TLS/SMTPS on port 465 (if supported) is recommended,
-;; otherwise STARTTLS on port 587 should be used.
-;HOST =
+;; Mail server protocol. One of "smtp", "smtps", "smtp+startls", "smtp+unix". If your provider does not explicitly say which protocol it uses but does provide a port, you can set SMTP_PORT instead and this will be inferred.
+;PROTOCOL =
+;
+;; Mail server address, e.g. smtp.gmail.com. For smtp+unix, this should be a path to a unix socket instead.
+;SMTP_ADDR =
 ;;
-;; Disable HELO operation when hostnames are different.
-;DISABLE_HELO =
+;; Mail server port. Common ports are 25 (insecure SMTP), 465 (SMTP Secure), and 587 (StartTLS); if no protocol is specified, it will be inferred by this setting.
+;SMTP_PORT =
+;;
+;; Enable HELO operation. If you don't know what this is, leave it blank.
```

Comment:
> If you don't know what this is, leave it blank.

I think this should be left out. 
See relevant discussion: https://github.com/go-gitea/gitea/pull/17440#discussion_r752890770

Also could the default value be added for this key?